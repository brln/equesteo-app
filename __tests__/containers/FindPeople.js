import configureStore from 'redux-mock-store'
import React from "react"
import {fromJS, Map} from "immutable"
import renderer from "react-test-renderer"
import { Provider } from 'react-redux'
import thunk from "redux-thunk"

import FindPeopleContainer from '../../containers/FindPeople'


const mockStore = configureStore([thunk])

test('render', () => {
  const store = mockStore(fromJS({
    pouchRecords: {},
    localState: {
      userSearchResults: {},
    }
  }))

  const tree = renderer.create(
    <Provider store={store}>
      <FindPeopleContainer />
    </Provider>)
  expect(tree.toJSON()).toMatchSnapshot()
})

