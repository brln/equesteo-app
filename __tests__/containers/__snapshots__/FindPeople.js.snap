// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<View
  style={
    Object {
      "alignItems": "stretch",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "flex-start",
      "width": "100%",
    }
  }
>
  <RCTScrollView
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View>
      <View
        onLayout={[Function]}
      >
        <View
          iosBarStyle="light-content"
          searchBar={true}
          style={
            Array [
              Object {
                "backgroundColor": "#F8F8F8",
                "borderBottomColor": "#a7a6ab",
                "borderBottomWidth": 0.5,
                "elevation": 3,
                "flexDirection": "row",
                "height": 64,
                "justifyContent": "center",
                "left": 0,
                "paddingLeft": 6,
                "paddingRight": 10,
                "paddingTop": 18,
                "right": 0,
                "shadowColor": undefined,
                "shadowOffset": undefined,
                "shadowOpacity": undefined,
                "shadowRadius": undefined,
                "top": 0,
              },
              Object {
                "backgroundColor": "#5bc0eb",
              },
            ]
          }
        >
          <View
            accessible={true}
            clickable={false}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "center",
                "backgroundColor": "white",
                "borderBottomWidth": 1,
                "borderColor": "transparent",
                "borderLeftWidth": 0,
                "borderRadius": undefined,
                "borderRightWidth": 0,
                "borderTopWidth": 0,
                "borderWidth": undefined,
                "flex": 1,
                "flexDirection": "row",
                "height": 40,
                "justifyContent": "flex-start",
                "marginLeft": 2,
                "opacity": 1,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": undefined,
                    "fontSize": 12,
                  },
                  Array [
                    Object {
                      "color": "#000",
                      "fontSize": 30,
                    },
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "transparent",
                      "color": "#414142",
                      "fontSize": 20,
                      "marginTop": 2,
                      "paddingLeft": 10,
                      "paddingRight": 10,
                    },
                  ],
                  Object {
                    "fontFamily": "Ionicons",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
            >
              
            </Text>
            <TextInput
              allowFontScaling={true}
              autoFocus={true}
              editable={true}
              onChangeText={[Function]}
              onSubmitEditing={[Function]}
              placeholder="Search by name"
              placeholderTextColor="#575757"
              rejectResponderTermination={true}
              style={
                Object {
                  "alignSelf": "center",
                  "color": "#000",
                  "flex": 1,
                  "fontSize": 17,
                  "height": 30,
                  "lineHeight": null,
                  "paddingLeft": 5,
                  "paddingRight": 5,
                  "top": 1.5,
                }
              }
              underlineColorAndroid="rgba(0,0,0,0)"
            />
            <View
              accessible={true}
              clickable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": undefined,
                      "fontSize": 12,
                    },
                    Array [
                      Object {
                        "color": "#000",
                        "fontSize": 30,
                      },
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "transparent",
                        "color": "#414142",
                        "fontSize": 20,
                        "marginTop": 2,
                        "paddingLeft": 10,
                        "paddingRight": 10,
                      },
                    ],
                    Object {
                      "fontFamily": "Ionicons",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
              >
                
              </Text>
            </View>
          </View>
        </View>
      </View>
      <RCTScrollView
        containerStyle={
          Object {
            "marginTop": 0,
          }
        }
        data={Object {}}
        disableVirtualization={false}
        getItem={[Function]}
        getItemCount={[Function]}
        horizontal={false}
        initialNumToRender={10}
        keyExtractor={[Function]}
        maxToRenderPerBatch={10}
        numColumns={1}
        onContentSizeChange={[Function]}
        onEndReachedThreshold={2}
        onLayout={[Function]}
        onMomentumScrollEnd={[Function]}
        onScroll={[Function]}
        onScrollBeginDrag={[Function]}
        onScrollEndDrag={[Function]}
        removeClippedSubviews={false}
        renderItem={[Function]}
        scrollEventThrottle={50}
        stickyHeaderIndices={Array []}
        updateCellsBatchingPeriod={50}
        viewabilityConfigCallbackPairs={Array []}
        windowSize={21}
      >
        <View />
      </RCTScrollView>
    </View>
  </RCTScrollView>
</View>
`;
